content/participate/connect.adoc

```diff
@@ -42,8 +42,7 @@ Here are the key ones:
 * link:https://community.jenkins.io/[community.jenkins.io] - our new community forum on Discourse.
   Jenkins end users, developers and contributors are welcome!
 
-Note that language groups and local communities might be using separate channels.
-See link:https://community.jenkins.io/c/language-groups/14[Language Groups on Discourse] for some references.
+See link:https://community.jenkins.io/t/all-communication-channels-that-we-are-aware-of/79[All Communication Channels that Jenkins has] for some references.
```

Comment:
```suggestion
Refer to link:https://community.jenkins.io/t/all-communication-channels-that-we-are-aware-of/79[the list of communication channels that Jenkins has] for further information.
```
Just a small suggestion to replace "see" with "refer to" and adjust the text accordingly.